Formula/commandbox.rb

```diff
@@ -1,16 +1,16 @@
 class Commandbox < Formula
   desc "CFML embedded server, package manager, and app scaffolding tools"
   homepage "https://www.ortussolutions.com/products/commandbox"
-  url "https://downloads.ortussolutions.com/ortussolutions/commandbox/3.8.0/commandbox-bin-3.8.0.zip"
-  sha256 "e184e380713b6e4e94c1266bf4abc352da2e5c7b19bdb9d5ce5f36665d1c58ec"
+  url "https://downloads.ortussolutions.com/ortussolutions/commandbox/3.9.0/commandbox-bin-3.9.0.zip"
+  sha256 "9497a7db8879150548b05d28594cc7d316556fc3a2e4602d3795b589ba85105e"
 
   bottle :unneeded
 
-  depends_on :java => "1.7+"
+  depends_on :java => "1.7+ < 9"
```

Comment:
you'd need to require Java 8 specifically and use a wrapper script

Reply:
@ilovezfs Can you explain what a "wrapper script" means?  I just need to allow either Java 7 or 8, but not 9.  I looked and looked on Google but couldn't find the correct syntax for this.  I couldn't even locate the Homebrew docs that covered the valid expressions above.  I just guessed with what I typed and it seemed to make the builds pass.